electroncash/commands.py

```diff
@@ -175,6 +177,58 @@ def commands(self):
         """List of commands"""
         return ' '.join(sorted(known_commands.keys()))
 
+    @command('n')
+    def getinfo(self):
+        """ network info """
+        net_params = self.network.get_parameters()
+        response = {
+            'path': self.network.config.path,
+            'server': net_params[0],
+            'blockchain_height': self.network.get_local_height(),
+            'server_height': self.network.get_server_height(),
+            'spv_nodes': len(self.network.get_interfaces()),
+            'connected': self.network.is_connected(),
+            'auto_connect': net_params[4],
+            'version': PACKAGE_VERSION,
+            'wallets': {k: w.is_up_to_date()
```

Comment:
To stay closer to upstream, how about also leaving the `'default_wallet': self.config.get_wallet_path(),` between `version` and `wallets`? I'm deserializing the JSON into a struct with `default_wallet`, so it would be nice if Electron Cash included it too.